BenadrylAutomation/testng.xml

```diff
@@ -13,6 +13,8 @@
 			<class name="com.jnj.testcases.ComparePageTest" /> 
 			<class name="com.jnj.testcases.FooterPageTest" />
 			<class name="com.jnj.testcases.OurIngredientsPageTest" />
+			<class name="com.jnj.testcases.SafetyInformationPageTest" />
+			
```

Comment:
@vaibhavNagvekarSJI Please remove the blank line

Reply:
fixed

Reply:
Verified @vaibhavNagvekarSJI